FirebaseAuth/Sources/Swift/MultiFactor/Phone/PhoneMultiFactorInfo.swift

```diff
@@ -51,8 +51,7 @@ import Foundation
 
     private let kPhoneNumberCodingKey = "phoneNumber"
 
-    private static var secureCodingWorkaround = true
-    override open class var supportsSecureCoding: Bool { return secureCodingWorkaround }
+    override public class var supportsSecureCoding: Bool { return true }
```

Comment:
nit: A stored property seems a little cleaner 
```suggestion
    override public class var supportsSecureCoding: Bool = true
```

Reply:
![Screenshot 2024-01-12 at 9 57 14 AM](https://github.com/firebase/firebase-ios-sdk/assets/73870/bd0e11ee-518e-4efa-9710-3cde756d1118)